packages/playwright-config-commons-ui/index.js

```diff
@@ -0,0 +1,35 @@
+const { devices } = require("@playwright/test");
+
+const config = {
+  timeout: 30 * 1000,
+  testMatch: /.*\.spec.js/,
+  reporter: "html",
+  use: {
+    actionTimeout: 0,
+    headless: false,
+    viewport: { width: 1280, height: 720 },
+    trace: "on-first-retry",
+  },
```

Comment:
```suggestion
  use: {
    trace: "on-first-retry",
  },
```

We can add other config if/when needed.